addons_rksv/pos_rksv/views/res_company.xml

```diff
@@ -9,6 +9,9 @@
             <field name="arch" type="xml">
                 <notebook position="inside">
                     <page string="BMF" attrs="{'invisible': [('rksv_at', '=', False)]}">
```

Comment:
Hiding a page where you have the checkbox inside the the hidden page? Sounds like a dead end.